### DISCUSSION TITLE
Dynamic Data Source configuration missing from Search Box webpart?

### DISCUSSION THREAD

**Discussion by ConaxLiu** [microsoft-search/pnp-modern-search#2988] [Q&A] [answered]
I was trying to follow the steps in the '[Create a search page with verticals](https://microsoft-search.github.io/pnp-modern-search/scenarios/Create-a-search-page-with-verticals-on-different-pages/)' scenario, where it says to configure the search box webpart to use dynamic data source. However, I do not see this configuration option in the web part.

Was this option removed from the webpart and therefore the scenario is outdated, or...?

**Comment by wobba** [comment, marked as answer]
Web part pane page 3.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/51104/a4fc0843-638a-4951-8b54-bcaf8548d0eb)

**Reply by ConaxLiu** [reply]
Ah...page 3!! Thank you! With only a few settings displayed on page 1, I didn't expect there would even be a page 2!
